Src/Support/Google.Apis.Auth/OAuth2/GoogleAuthConsts.cs

```diff
@@ -73,20 +73,22 @@ public static class GoogleAuthConsts
         /// <summary>Installed application localhost redirect URI.</summary>
         public const string LocalhostRedirectUri = "http://localhost";
 
+        internal const string IamServiceHostPrefix = "https://iamcredentials.";
+
         /// <summary>IAM access token endpoint for service account.</summary>
-        internal const string IamServiceAccountEndpointCommonPrefix = "https://iamcredentials.googleapis.com/v1/projects/-/serviceAccounts/";
+        internal const string IamServiceAccountEndpointCommonPrefixFormat = $"{IamServiceHostPrefix}{{0}}/v1/projects/-/serviceAccounts/";
 
         /// <summary>IAM access token verb.</summary>
         internal const string IamAccessTokenVerb = "generateAccessToken";
 
         /// <summary>IAM access token endpoint format string. To use it insert the service account email.</summary>
-        internal static readonly string IamAccessTokenEndpointFormatString = $"{IamServiceAccountEndpointCommonPrefix}{{0}}:{IamAccessTokenVerb}";
+        internal static readonly string IamAccessTokenEndpointFormatString = $"{IamServiceAccountEndpointCommonPrefixFormat}{{1}}:{IamAccessTokenVerb}";
```

Comment:
We can make all of these const, I believe. (Before .NET 6, we couldn't - but IamServiceAccountEndpointCommonPrefixFormat proves that we can use const interpolated strings.)